tests/roles/ironic_adoption/defaultes/main.yaml

```diff
@@ -0,0 +1,75 @@
+---
+ironic_patch: |
+  spec:
+    ironic:
+      enabled: true
+      template:
+        rpcTransport: oslo
+        databaseInstance: openstack
+        ironicAPI:
+          replicas: 1
+          override:
+            service:
+              internal:
+                metadata:
+                  annotations:
+                    metallb.universe.tf/address-pool: internalapi
+                    metallb.universe.tf/allow-shared-ip: internalapi
+                    metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                spec:
+                  type: LoadBalancer
+        ironicConductors:
+        - replicas: 1
+          networkAttachments:
+          - baremetal
+          provisionNetwork: baremetal
+          storageRequest: 10G
+          storageClass: {{ storage_class_name }}
```

Comment:
We may need to add a default for storage_class_name too, to make sure it is already defined when the ironic_patch value is being processed. I think when the template was in `tasks.yaml` it relied on the variable being included from OVN role defaults, but that may not be a reliable way anymore. We should probably copy the default from the OVN file into this file too. https://github.com/openstack-k8s-operators/data-plane-adoption/blob/eca663af5a80df64a6b78388ce5a7d9181615fa0/tests/roles/ovn_adoption/defaults/main.yaml#L2